source/FAQ/index.rst

```diff
@@ -286,7 +286,7 @@ How do I change my default login shell?
 
 To change your default shell:
 
-* Log into AIM.
+* Log into AIM https://aim.rdhpcs.noaa.gov/
```

Comment:
Could you format it like the below?  

```
Log into `AIM  <https://aim.rdhpcs.noaa.gov/>`_.
```

This will use "AIM" as a hyperlink to the URL.  (Looks cleaner that way, and is more consistent with links elsewhere in the docs)